src/mkindex_algo.hpp

```diff
@@ -125,7 +123,7 @@ loadSubjSeqsAndIds(LambdaIndexerOptions const & options)
             extractAccIds(id, count);
 
         if (options.truncateIDs)
-            ids.push_back(id | seqan3::view::take_until(seqan3::is_space));
+            ids.push_back(id | seqan3::view::take_until(seqan3::is_space) | std::ranges::to<std::string>);
```

Comment:
Why?

Reply:
Views are no longer implicitly convertible to containers, you need to pipe them into `ranges::to` to convert.